packages/twenty-website/src/content/developers/frontend-development/hotkeys.mdx

```diff
@@ -144,7 +144,7 @@ As an example, the hotkey scopes for each page are defined in the `PageHotkeySco
 ```tsx
 export enum PageHotkeyScope {
   Settings = 'settings',
-  CreateWokspace = 'create-workspace',
+  CreateWorkspace = 'create-workspace',
```

Comment:
I guess we have the same typo in twenty-front

Reply:
I'll check this

Reply:
There is, I'll create a PR with more typo fixes soon